styles.css

```diff
@@ -0,0 +1,15 @@
+body {
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    margin: 100px;
+    font-size: 18px;
+}
+
+
+img {
+    height: 300px;
+    width: 450px;
+    display: flex;
+    margin: 30px auto;
+}
```

Comment:
Not sure if you have covered CSS classes yet. If you have I make a class like this 

```css
img.standard-image {
     height: 300px;
     width: 450px;
     display: flex;
     margin: 30px auto;
 }
```

and then the tag becomes
```html
<img src="Ramen1.jpeg" class="standard-image" alt="Vegetable Ramen with runny half-boiled egg on top">
```